StPicoDstMaker/StPicoDst.h

```diff
@@ -21,8 +21,9 @@ class StPicoMtdPidTraits;
 class StPicoDst : public TObject
 {
 public:
-  /// constructor
-  StPicoDst();
+  StPicoDst() {}
+  virtual ~StPicoDst() {}
```

Comment:
Hm... I doubt there is anything polymorphic about a collection of static members in this class :) Also, nothing inherits from `StPicoDst` AFAIK, and we probably should not expect any inheritance in the future. At least right now I can't come up with a case where one would like to inherit from StPicoDst.

BTW, in the next commit you remove TObject as a base and that is very good 👍  because indeed it seems to be unnecessary. If I remember correctly StPicoDst does not own any resources so, in principle, an explicit destructor is also not necessary.

Below in this file you still have one more virtual method `virtual void Print(Option_t* option = "") const;` that can be converted to a non-virtual one as there no need to use the same signature as in TObject.